File: src/Frontend/src/app/services/models/product.dto.ts
export interface ProductDto {
    id: number;
    name: string;
    calories: number;
    type: TypeofFood;
    unit: unitType; // enum измерение
}

enum TypeofFood {
    meat,
    drink,
    cereal,
    vegetable,
    fruit,
    confection,
    bake
}

enum unitType {
    Milliliters,
    Gram,
    Pieces

}

File: src/Web/WeightControl.BusinessLogic/Models/ProductDto.cs
using System;
namespace WeightControl.BusinessLogic.Models
{
    public class ProductDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Calories { get; set; }
        public int Type { get; set; }
        public int Unit { get; set; }
    }
}
